docs/docs/behaviour.md

```diff
@@ -8,4 +8,12 @@ title: Behaviour
 3. After returning from the authentication provider, the oauth tokens are stored in the configured session store (cookie, redis, ...) and a cookie is set
 4. The request is forwarded to the upstream server with added user info and authentication headers (depending on the configuration)
 
+The log in interface will, by default, redirect back to the configured domain with the user's original path.
```

Comment:
Wondering if this is the right place to put this documentation? Maybe instead it should be in the configuration section since the behaviour is a high level overview of how the project works, and this seems to be going into more low level documentation.

I think we already have some documentation for using nginx with OAuth2 Proxy, so perhaps it can go there?